tests/test_reporting.py

```diff
@@ -118,13 +115,16 @@ def test_generate_report_dict_script(self):
             # Strip trailing newlines.
             self.assertEqual(source_line.rstrip(), regression_line.rstrip())
 
-    @unittest.skip("skipping due to different number truncation in py36 and py27")
+    @unittest.skip(
+        "skipping due to different number truncation in py36 and py27")
     def test_generate_report_shape_json(self):
         """Reporting: testing full report w/ shape table data and json file."""
         from natcap.invest import reporting
 
         workspace_dir = self.workspace_dir
-        workspace_dir = 'C:/Users/dmf/projects/invest_dev/py36_compatibility/reporting'
+        workspace_dir = os.path.join(
+            'C:', os.sep, 'Users', 'dmf', 'projects', 'invest_dev',
+            'py36_compatibility', 'reporting')

```

Comment:
@phargogh , good catch. I don't have much memory of this, but it sounds like the expected html differed slightly between python versions.  Given how little use there is of the `reporting` module, I'm very okay with removing the test entirely. We've been living without if for a while anyway by skipping it. Otherwise, I guess we unskip the test and update the regression data so that it passes on python3.

Reply:
I removed it.

Reply:
Removed here: 4961598ed7b354b751e9009dc7864f757be9cedf